app/controllers/api/v1/gc_organisations_controller.rb

```diff
@@ -28,6 +28,19 @@ def names
       find_record_and_render_json(organisation_name_serializer)
     end
 
+    api :GET, '/v1/organisations/:id/organisation_orders', "List all orders associated with organisation"
```

Comment:
I realise I'm a little late to the party here but normally the API url would be more like this

`/v1/organisations/1/orders`

Reply:
Sure.. @steveyken i'll this change in separate PR.